zendesk_api.gemspec

```
@@ -13,6 +13,14 @@ Gem::Specification.new do |s|
   s.description = 'Ruby wrapper for the REST API at https://www.zendesk.com. Documentation at https://developer.zendesk.com.'
   s.license     = 'Apache License Version 2.0'
 
+  s.metadata = {
+    'bug_tracker_uri' => 'https://github.com/zendesk/zendesk_api_client_rb/issues',
+    'changelog_uri' => 'https://github.com/zendesk/zendesk_api_client_rb/blob/master/CHANGELOG.md',
+    'documentation_uri' => 'https://zendesk.github.io/zendesk_api_client_rb/',
```

I notice you host your own RDoc. Alternatively, use the automatically generated RDoc at www.rubydoc.info.

```suggestion
    'documentation_uri' => "https://www.rubydoc.info/gems/zendesk_api/#{s.version}",
```

---

I think we should use the rubydoc instead of the hosted one, which hasn't been updated for the last versions... what do you think @ggrossman ?

---

Sounds good!